addons/spreadsheet_dashboard_im_livechat/data/files/livechat_dashboard.json

```diff
@@ -250,14 +250,14 @@
             "type": "scorecard",
             "background": "#FEF2F2",
             "baseline": "Data!E3",
-            "baselineDescr": "sec",
+            "baselineDescr": "sec last period",
             "keyValue": "Data!D3",
             "humanize": true
           }
         },
         {
           "id": "5b8972f1-db5f-48dd-aeca-c0bc87c7155d",
-          "x": 680,
+          "x": 770,
```

Comment:
I personally don't care that much what you do with your dashboards, however there is a spreadsheet formula called `format.large.number( number_to_format )`. It takes an arbitrary large number and formats it in a way to never show more than 7 characters
![image](https://github.com/user-attachments/assets/086cd248-e494-40a6-b18f-d79a33b40dd5)